src/theme/material/tooltip.m.css

```diff
@@ -1,2 +1,38 @@
+@import './variables.css';
+
+:root {
+	--mdc-tooltip-font-size: 0.975rem;
```

Comment:
i think this should be in the variables file

Reply:
Moved this to a var in `variables.css`